missioncontrol/etl/versions.py

```diff
@@ -0,0 +1,23 @@
+import requests
+from django.core.cache import cache
+
+from missioncontrol.settings import FIREFOX_VERSION_URL
+
+
+def get_firefox_versions():
+    firefox_versions = cache.get('firefox_versions')
+    if firefox_versions:
+        return firefox_versions
+
+    # map the version api's to telemetry channel names
+    r = requests.get(FIREFOX_VERSION_URL)
```

Comment:
Can you please set a timeout for this request and handle he case when the json deserialization fails?